distgen/README.md

```diff
@@ -43,3 +43,14 @@ global variables to generate. (See an example of a JSON file at
 package dist
 //go:generate go run github.com/chef/go-libs/distgen global.go dist https://example.com/path/to/glob_dist.json
 ```
+
+### Using an Environment Variable
```

Comment:
Documentation YESS!! Thank you for that, do you think you could update the [doc.go](https://github.com/chef/go-libs/blob/master/distgen/doc.go) as well so that we have this documentation matching our [godoc.org](https://godoc.org/github.com/chef/go-libs/distgen)

Reply:
Of course.  I will look at performing the addition tonight.